Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: astronomer
-version: 0.28.0
-appVersion: 0.28.0
+version: 0.28.1
+appVersion: 0.28.1
 description: Helm chart to deploy the Astronomer Platform
```

Comment:
We are already at 0.28.2 now. This should be 0.28.3

Reply:
should this be done in main or just release branch?

Reply:
@vishwas-astro can you please help here? I'm not sure that i understand @danielhoherd